_posts/cn/2018-05-17-easy-build-microservice-system-part-I.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "轻松微服务系列：使用脚手架功能一键构建微服务"
+lang: cn
+ref: easy-build-microservice-system-part-I
+permalink: /cn/docs/easy-build-microservice-system-part-I/
+excerpt: "轻松微服务系列：使用脚手架功能一键构建微服务"
+last_modified_at: 2018-05-17T19:05:00+08:00
+author: Yangyong Zheng
+tags: [Archetypes, Scaffold]
+redirect_from:
+  - /theme-setup/
+---
+
+## 轻松微服务系列：使用脚手架功能一键构建微服务
+提到微服务，相信“程序猿”们已经不再陌生，估计大家都多少尝试过微服务框架，也遇到了不少“坑”直呼坑爹吧？实际上微服务的初衷正是为了“解放”开发者，使得系统构建的过程更加轻松愉快，所以我们开了这个“轻松微服务系列”，和大家一起愉快的走一遭！废话不多说，上菜：
+
+### 牛刀小试
```

Comment:
How about change the title to obviously pointed out that one-click build micro service

Reply:
Done